lib/runtime/src/system_status_server.rs

```diff
@@ -185,7 +207,8 @@ async fn health_handler(state: Arc<SystemStatusState>) -> impl IntoResponse {
     let response = json!({
         "status": healthy_string,
         "uptime": uptime,
-        "endpoints": endpoints
+        "endpoints": endpoint_details,
+        "has_active_monitoring": has_health_checks
```

Comment:
not sure we need this - just the endpoint details - maybe with last checked time?